storage/persistent-storage/persistent-storage-snapshots.adoc

```diff
@@ -0,0 +1,24 @@
+[id="persistent-storage-snapshots"]
+= Persistent storage using volume snapshots
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-snapshots
+
+toc::[]
+
+This document describes how to use `VolumeSnapshots` to protect against data loss in {product-title}. Familiarity with xref:../storage/understanding-persistent-storage.adoc#persistent-volumes_understanding-persistent-storage[persistent volumes] is suggested.
```

Comment:
This xref should be:

> xref:../../storage/understanding-persistent-storage.adoc#persistent-volumes_understanding-persistent-storage[persistent volumes]

Note the additional `../` at the beginning. This is because we must begin from the root directory, and this assembly is nested in two levels of subdirectories.